01_prepare_host.sh

```diff
@@ -38,20 +38,31 @@ if ! command -v kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
-# Download Ironic binary and Ironic inspector image
-
-mkdir -p "$IRONIC_DATA_DIR/html/images"
-pushd "$IRONIC_DATA_DIR/html/images"
-if [ ! -f ironic-python-agent.initramfs ]; then
-    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/current-tripleo-rdo/ironic-python-agent.tar | tar -xf -
-fi
+# Clean-up any old ironic containers
+for name in ironic ironic-inspector dnsmasq httpd mariadb ipa-downloader; do
+    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name
+    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f
+done
 
-for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE ; do
+# Clean-up existing pod, if podman
+case $CONTAINER_RUNTIME in
+podman)
+  if  sudo "${CONTAINER_RUNTIME}" pod exists ironic-pod ; then
+      sudo "${CONTAINER_RUNTIME}" pod rm ironic-pod -f
+  fi
+  sudo "${CONTAINER_RUNTIME}" pod create -n ironic-pod
+  ;;
+esac
+
+# Pull images we'll need
+for IMAGE_VAR in IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE; do
```

Comment:
This is missing ironic image, as it is used for httpd container

Reply:
Fixed